AndroidPayPalWebSdkIntegrationExample/app/src/main/java/com/example/androidpaypalwebsdkintegrationexample/OnPaymentCompletionActivity.java

```diff
@@ -0,0 +1,54 @@
+package com.example.androidpaypalwebsdkintegrationexample;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.TextView;
+
+import com.example.androidpaypalwebsdkintegrationexample.databinding.ActivityOnPaymentCompletionBinding;
+
+import org.json.JSONObject;
+
+/**
+ * This is a sample activity which demonstrates an example payment success activity page.
+ * This Activity should be invoked after the buyer has reviewed and approved their payment
+ * on web application hosted by merchant.
+ * The web application should use payment methods like PayPal, Braintree to collect payments from buyer
+ */
+public class OnPaymentCompletionActivity extends AppCompatActivity {
+    private ActivityOnPaymentCompletionBinding binding;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_on_payment_completion);
+        binding = ActivityOnPaymentCompletionBinding.inflate(getLayoutInflater());
+        // ATTENTION: This was auto-generated to handle app links.
+        Intent appLinkIntent = getIntent();
+        String appLinkAction = appLinkIntent.getAction();
+        Uri appLinkData = appLinkIntent.getData();
```

Comment:
We can probably omit this generated code since we're not doing anything with these variables.